src/main/java/org/valdroz/vscript/EquationParser.java

```diff
@@ -454,13 +455,18 @@ private BaseNode parseAsIdentifiable() {
             int mathFuncOpCode = determineBuildinFuncCode(word);
 
             if (isFunction) {
-                if (mathFuncOpCode > 0) {
+                if (mathFuncOpCode > 0 &&  mathFuncOpCode != 170){
```

Comment:
`&& mathFuncOpCode != 170` is not required

Reply:
removed condition `&& mathFuncOpCode != 170`